backend/README.md

```diff
@@ -90,6 +90,12 @@ backend is running:
 ```bash
 # in main folder while docker-compose is running
 $ docker exec backend yarn run db:migrate init
+
+# only once: init admin user and create indexes and constraints in Neo4j database
+# for development
+$ docker compose exec backend yarn prod:migrate init
+# in production mode use command
+$ docker compose exec backend /bin/sh -c "yarn prod:migrate init"
```

Comment:
how is this section related to apple m1?

Reply:
I move the command from the deleted Apple M1 section to a fitting place to have it in the docu.